I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 36, in script call
    call office_floor_2 from _call_office_floor_2
  File "game/office_floor_2.rpy", line 759, in script
    $ addRockyhealth(-1)
  File "game/office_floor_2.rpy", line 759, in <module>
    $ addRockyhealth(-1)
NameError: name 'addRockyhealth' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 36, in script call
    call office_floor_2 from _call_office_floor_2
  File "game/office_floor_2.rpy", line 759, in script
    $ addRockyhealth(-1)
  File "C:\Games\renpy-8.3.4-sdk\renpy\ast.py", line 827, in execute
    renpy.python.py_exec_bytecode(self.code.bytecode, self.hide, store=self.store)
  File "C:\Games\renpy-8.3.4-sdk\renpy\python.py", line 1178, in py_exec_bytecode
    exec(bytecode, globals, locals)
  File "game/office_floor_2.rpy", line 759, in <module>
    $ addRockyhealth(-1)
NameError: name 'addRockyhealth' is not defined

Windows-10-10.0.26100 AMD64
Ren'Py 8.3.4.24120703
Cafe of The Dead 1.0
Mon Mar 31 07:45:58 2025
